tests/chainer_tests/functions_tests/math_tests/test_sqrt.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+import chainer.functions as F
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+class UnaryFunctionsTestBase(unittest.TestCase):
```

Comment:
Yes, because test for rsqrt function is implemented with this base class in another PR.